lib/mix/tasks/secrex.encrypt.ex

```diff
@@ -14,11 +14,13 @@ defmodule Mix.Tasks.Secrex.Encrypt do
     key = encryption_key()
 
     for path <- secret_files() do
-      Mix.shell().info("encrypting #{path}")
-      encrypted = encrypt(path, key)
+      Mix.shell().info("Encrypting #{path}")
+
+      {:ok, encrypted} = encrypt(path, key)
+
       File.write!(encrypted_path(path), encrypted)
     end
 
-    Mix.shell().info("files encrypted")
+    Mix.shell().info("Files encrypted")
```

Comment:
```suggestion
    Mix.shell().info("Files have been encrypted")
```
To match the counterpart message?